chia/full_node/coin_store.py

```diff
@@ -85,6 +82,13 @@ async def create(cls, db_wrapper: DBWrapper, cache_size: uint32 = uint32(60000))
         self.coin_record_cache = LRUCache(cache_size)
         return self
 
+    async def num_unspent(self) -> int:
+        async with self.coin_record_db.execute("SELECT COUNT(*) FROM coin_record WHERE spent_index=0") as cursor:
```

Comment:
Not an expert in DB performance or if this even matters, but there is a `spent` field on the coin_record too that seems to be the natural tool for the job here.

Reply:
Introduced in https://github.com/Chia-Network/chia-blockchain/pull/9780 if you want to follow up on the concerns.